tests/policy_engine/resources/policies/slsa_verifier.yaml

```diff
@@ -10,10 +10,6 @@ definition:
   predicateType: https://slsa.dev/provenance/v0.2
 
   predicate:
-    builder:
```

Comment:
Why is this part removed in the policy?

Reply:
Because the YAML policy doesn't support regex, so we need to keep adding exact tag versions for the builder, e.g.,
` https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v1.2.0`